packages/analytics/src/errors.ts

```diff
@@ -39,12 +42,18 @@ const ERRORS: ErrorMap<AnalyticsError> = {
     'settings() must be called before initializing any Analytics instance' +
     'or it will have no effect.',
   [AnalyticsError.INTEROP_COMPONENT_REG_FAILED]:
-    'Firebase Analytics Interop Component failed to instantiate'
+    'Firebase Analytics Interop Component failed to instantiate',
+  [AnalyticsError.INDEXED_DB_UNSUPPORTED]:
+    'IndexedDB is not supported by current browswer',
+  [AnalyticsError.INVALID_INDEXED_DB_CONTEXT]:
+    "Environment doesn't support IndexedDB functionality: {$errorInfo}",
+  [AnalyticsError.COOKIE_NOT_ENABLED]: 'Browser cookie not enabled'
```

Comment:
Suggest "Cookies not enabled in this browser environment. Analytics requires cookies to be enabled." or something similar?